sirepo/job_supervisor.py

```diff
@@ -284,16 +284,21 @@ def __db_init(self, req, prev_db=None):
             history=self.__db_init_history(prev_db),
             isParallel=c.isParallel,
             isPremiumUser=c.get('isPremiumUser'),
-            jobRunMode=c.jobRunMode,
+            jobRunMode=c.get(
+                'jobRunMode',
+                self.db.jobRunMode if 'db' in self else None,
```

Comment:
This is a little funky and deserves a comment, perhaps.

Reply:
I added an assert and comment.

Reply:
And changed the behavior slightly. After more thought we should never run into the None case.